### PR TITLE
feat(databases): add maintenance

### LINKED ISSUE #329: Add documentation about maintenance

### PR DESCRIPTION
**Pull request by SCedricThomas**
fix #329

### CODE REVIEW

**Review by yohann-bacha — COMMENTED** [review]
well if the api has indeed a "maintenance" without plural, but containing multiple maintenances, this is not coherent. but well if it is already implemented like that everywhere, well we'll gonna stay like this

File: _database_api/maintenance.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+curl -H "Accept: application/json" -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $DB_BEARER_TOKEN" \
+  -X GET https://$DB_API_URL/api/databases/my-db-123/maintenances
+```
+
+Returns 200 OK
+
+```json
+{
+  "maintenance": [
```
[the remark this answers is not in the document]
Comment (SCedricThomas) [reply _database_api/maintenance.md:62]: maintenance common plural form has no 's'

[the remark this answers is not in the document]
Comment (SCedricThomas) [reply _database_api/maintenance.md:62]: https://www.wordhippo.com/what-is/the-plural-of/maintenance.html

**Comment by SCedricThomas** [comment]
Screenshot from 2023-07-17 11-27-11
Screenshot from 2023-07-17 11-27-17

File: _database_api/maintenance.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  }
+}
+```
+--- row ---
+
+## List database Maintenance
+
+--- row ---
+
+`GET https://$DB_API_URL/api/databases/[:db_id]/maintenances?page=[:page_number]&per_page=[:per_page_number]`
```
Comment (curzolapierre) [inline _database_api/maintenance.md:44]: We don't usually display the pagination parameter. Instead, you can do the same as for the logs-archive, for example. By specifying that this request supports the pagination system.

File: _database_api/maintenance.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Retrieve information of a specific maintenance.
+
+||| col |||
+
+Example request
+
+```shell
+curl -H "Accept: application/json" -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $DB_BEARER_TOKEN" \
+  -X GET https://$DB_API_URL/api/databases/my-db-123/maintenances/my-maintenance-123
```
Comment (curzolapierre) [inline _database_api/maintenance.md:107]:
```suggestion
  -X GET https://$DB_API_URL/api/databases/my-db-123/maintenance/my-maintenance-123
```

**Review by curzolapierre — CHANGES_REQUESTED** [review]

**Review by curzolapierre — APPROVED** [review]

**Review by curzolapierre — APPROVED** [review]
Thanks for the update of pagination object !

**Review by yohann-bacha — APPROVED** [review]

**Pull request merged**